tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/kernels/pgm_dispatch_perf.cpp

```diff
@@ -3,8 +3,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // NULL kernel is not 0, subtract off overhead
-#if KERNEL_BYTES > 30
-uint8_t data1[KERNEL_BYTES-30] __attribute__ ((section ("l1_data"))) __attribute__((used));
+#if KERNEL_BYTES > 16
+uint8_t data1[KERNEL_BYTES-16] __attribute__ ((section ("l1_data_test_only"))) __attribute__((used));
```

Comment:
I don't fully understand the context for this subtraction. Why was it 30 before and 16 now?

Reply:
magic numbers bad :)

Reply:
unrelated.  that number was tuned to get the size kernel that was requested on the command line.  I think some of Nathan's work on the init asm shrunk the code, so now we need a few more bytes to reach the requested size